### PR TITLE
feat(idp): display metadata details

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

Implementation IdP Config displaying metadata

## Why

Enable Company/IT Admins to see their identity provider config inside the "Config" overlay if any configuration does already exist

## Issue

https://github.com/eclipse-tractusx/portal-frontend/issues/475

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

File: src/features/admin/idpApiSlice.ts [hunk 1]
```diff
@@ -136,15 +137,16 @@ export interface UserIdentityProviders {
...
-  redirectUrl: string
   enabled: boolean
+  identityProviderCategoryId: string
+  identityProviderId: string
+  identityProviderTypeId: IDPCategory
+  redirectUrl: string
   mappers: Array<IDPMapper>
   oidc?: OIDCType
-  saml?: OIDCType
+  saml?: SAMLType
```
Comment (jjeroch) [inline src/features/admin/idpApiSlice.ts:149]: @nidhigarg-bmw  can you provide me some details why this got changed

**Comment by jjeroch** [comment]
@Leon-Gr  please review the locales files (de & en)

File: src/features/admin/idpApiSlice.ts (hunk 1 above)
Comment (nidhigarg-bmw) [reply src/features/admin/idpApiSlice.ts:149]: @jjeroch In swagger this type was different so I managed to changed it accordingly
image

Comment (jjeroch) [reply src/features/admin/idpApiSlice.ts:149]: ok understood, thanks

**Review by manojava-gk — APPROVED** [review]
lftm

**Comment by nidhigarg-bmw** [comment]
@evegufy Can you please merge this PR ?

**Review by evegufy — APPROVED** [review]

**Pull request merged**